<!DOCTYPE html>
<html>
  <head>
    <title>Talks – Florian Wagner, PhD – Computational biology and single-cell RNA-Seq</title>

        <meta charset="utf-8" />
    <meta content='text/html; charset=utf-8' http-equiv='Content-Type'>
    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0'>

    
    <meta name="description" content="Computational biology and single-cell RNA-Seq">
    <meta property="og:description" content="Computational biology and single-cell RNA-Seq" />
    
    <meta name="author" content="Florian Wagner, PhD" />

    
    <meta property="og:title" content="Talks" />
    <meta property="twitter:title" content="Talks" />
    

    <!--[if lt IE 9]>
      <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <link rel="stylesheet" type="text/css" href="/test-page/style.css"/>
    <link rel="alternate" type="application/rss+xml" title="Florian Wagner, PhD - Computational biology and single-cell RNA-Seq" href="/test-page/feed.xml" />

    <!-- Created with Jekyll Now - http://github.com/barryclark/jekyll-now -->
  </head>

  <body>
    <div class="wrapper-masthead">
      <div class="container">
        <header class="masthead clearfix">
          <a href="/test-page/" class="site-avatar"><img src="/test-page/assets/dna-icon-2316641.svg" /></a>

          <div class="site-info">
            <h1 class="site-name"><a href="/test-page/">Florian Wagner, PhD</a></h1>
            <p class="site-description">Computational biology and single-cell RNA-Seq</p>
          </div>

          <nav>
 	    <a href="/test-page/">Blog</a>
	    <!--<a href="/test-page/projects">Projects</a>-->
	    <!--<a href="/test-page/publications">Publications</a>-->
            <a href="/test-page/talks">Talks</a>
            <a href="/test-page/about">About</a>
	    <!--<a href="/test-page/about">Contact</a>-->
          </nav>
        </header>
      </div>
    </div>

    <div id="main" role="main" class="container">
      <article class="page">

  <h1>Talks</h1>

  <div class="entry">
    <h3 id="nyu-school-of-medicine-abl-single-cell-analysis-club">NYU School of Medicine ABL Single-cell analysis club</h3>

<p>These are a series of talks that I was invited to give at the Single-cell Analysis Club, a monthly forum hosted by the Applied Bioinformatics Laboratories (ABL) at NYU School of Medicine. These talks took place in an informal setting, and I’m thankful for the great questions that I was being asked during different parts of each talk. I would like to thank Itai Yanai for the opportunity to speak at these meetings.</p>

<ul>
  <li>
    <p><a href="https://vimeo.com/344408348#t=4m39s">Denoising single-cell RNA-Seq data</a> (6/11/2019)
In this talk, I give an introduction into how single-cell RNA-Seq measurements are affected by noise, and provide a very brief overview of the different denoising methods that have been proposed. I also try to clear up common misconceptions about the characteristics of noise in scRNA-Seq data, for example regarding “dropouts”. Finally, I discuss our recently developed <a href="https://www.biorxiv.org/content/10.1101/655365v2">ENHANCE algorithm for denoising scRNA-Seq data</a>. Thank you to Dalia Barkley for providing an R demonstration of our ENHANCE algorithm.</p>
  </li>
  <li>
    <p><a href="https://vimeo.com/325281275#t=0m35s">Visualizing single-cell RNA-Seq data</a> (3/11/2019)
In this talk, I give an introduction to the two most popular visualization techniques for single-cell RNA-Seq data, t-SNE and UMAP. I try to answer some of the most commonly asked questions regarding these methods, using examples from real data. In the first part of the talk, I discuss t-SNE. Then, Dr. Pratip Chattopadhyay discusses the use of t-SNE for flow cytometry data. In the last part, Afterwards, I continue with a discussion of UMAP. Unfortunately, a minute or so from the beginning of my talk is missing, but the video still captures my entire discussion of t-SNE.</p>
  </li>
  <li>
    <p><a href="https://vimeo.com/315894191#t=50m45s">Clustering single-cell RNA-Seq data</a> (2/4/2019)
In this talk, I show some examples of applying standard clustering algorithms, specifically hierarchical clustering and k-means, to single-cell RNA-Seq data. To show that single-cell RNA-Seq data can be effectively clustered using only a few simple steps. This is the second half of a session on clustering single-cell RNA-Seq data (the first half was presented by Itai Yanai).</p>
  </li>
</ul>


  </div>
</article>

    </div>

    <div class="wrapper-footer">
      <div class="container">
        <footer class="footer">
          
<a href="mailto:florian.wagner@nyu.edu"><i class="svg-icon email"></i></a>


<a href="https://github.com/flo-compbio"><i class="svg-icon github"></i></a>




<a href="https://www.twitter.com/flo_compbio"><i class="svg-icon twitter"></i></a>



        </footer>
      </div>
    </div>

    

  </body>
</html>
